theseus/geometry/se2.py

```diff
@@ -44,7 +44,7 @@ def rand(
         *size: int,
         generator: Optional[torch.Generator] = None,
         dtype: Optional[torch.dtype] = None,
-        device: Optional[torch.device] = None,
+        device: Optional[theseus.constants._TheseusDevice] = None,
```

Comment:
Some files instead do `from theseus.constants import _TheseusDevice`. We could pick one for consistency.

Reply:
I was going to change all of them to the `from` version, but the files that have the full import are using things like `PI` and `EPS` and I thought it would just make the code really confusing for people to figure out where these are coming from. I thought clarity was better than consistency in this case.

Reply:
Yeah makes sense. Should we switch others to `theseus.constants._TheseusDevice` also?

Reply:
Too verbose, I think? I don't think it's a big deal if we are not fully consistent.